modules/library/src/main/scala/zio/elasticsearch/executor/HttpExecutor.scala

```diff
@@ -337,16 +337,15 @@ private[elasticsearch] final class HttpExecutor private (esConfig: ElasticConfig
     }
 
   private def executeSearch(r: Search): Task[SearchResult] = {
-    val body = r.sortBy match {
-      case sorts if sorts.nonEmpty =>
-        Obj(
-          List(
-            "query" -> r.query.paramsToJson(fieldPath = None),
-            "sort"  -> Arr(sorts.toList.map(_.paramsToJson): _*)
-          ): _*
-        )
-      case _ =>
-        r.query.toJson
+    val sortJson =
+      if (r.sortBy.isEmpty) None else Some(Obj("sort" -> Arr(r.sortBy.toList.map(_.paramsToJson): _*)))
+    val highlightsJson =
+      r.highlights.map(_.toJson)
+    val body = (sortJson, highlightsJson) match {
```

Comment:
Can you add a new line before? Also, maybe we can use `sort` and `highlights` and remove the `Json` suffix.